src/main/java/com/kodilla/ecommercee/controller/GroupController.java

```diff
@@ -0,0 +1,67 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.GroupDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/groups")
+@RequiredArgsConstructor
```

Comment:
What is the use of @RequiredArgsConstructor here?
GroupController has no atributes.